_includes/project-footer.html

```diff
@@ -17,8 +17,11 @@
     </div>
     <div class="width-3-12 width-12-12-m justify-self-center align-self-center">
       <p>
-        Strimzi is licensed under the<br/>
-        <a href="https://strimzi.io/LICENSE">Apache Licence, Version 2.0</a>
+        &copy; Strimzi Authors {{ 'now' | date: "%Y" }} | Documentation distributed under <a href="https://creativecommons.org/licenses/by/4.0">CC-BY-4.0</a>
```

Comment:
This now gives the website two copyrights - one for Strimzi Authors and another one for CNCF. Does that actually work? I would assume the page should have only one copyright, or? I can see that for example kubernetes.io has the same, but it still looks strange.